apps/minifront/vite.config.ts

```diff
@@ -8,7 +8,7 @@ import polyfillNode from 'vite-plugin-node-stdlib-browser';
 
 export default defineConfig(({ mode }) => {
   return {
-    define: { 'globalThis.__DEV__': mode === 'development' },
+    define: { 'globalThis.__DEV__': mode !== 'production' },
```

Comment:
During a test run, the mode is apparently `test`. So inverting the conditional so it can catch this mode as well. Otherwise, some tests fail.